### PR TITLE
feat(otellogs): update configuration according to the newest version

### PR DESCRIPTION
**Pull request by sumo-drosiek**
##### Description

feat(otellogs): update configuration according to the newest version

- enable multipart for docker
- add multipart for CRI logs

---

##### Checklist

Remove items which don't apply to your PR.

- [ ] Changelog updated

###### Testing performed

- [ ] Redeploy fluentd and fluentd-events pods
- [ ] Confirm events, logs, and metrics are coming in

### CODE REVIEW

File: deploy/helm/sumologic/values.yaml [hunk 1]
```diff
@@ -4370,37 +4370,42 @@ otellogs:
...
           # Output Body (JSON): { "log": "2001-02-03 04:05:06 first line\n  second line\n  third line\n", "stream": "stdout" }
-          # - id: merge-multiline-logs
-          #   type: recombine
-          #   combine_field: log
-          #   is_first_entry: $$body.log matches "^[^\\s]"
+          - id: merge-multiline-logs
+            type: recombine
+            output: extract-metadata-from-filepath
+            combine_field: log
+            is_first_entry: $$body.log matches "^[^\\s]"
```
Comment (sumo-drosiek) [inline deploy/helm/sumologic/values.yaml:4408]: @swiatekm-sumo I just uncommented this, but my question is, why is this different from this what we do in fluent-bit?
(in fluent bit we matches by date format)

File: deploy/helm/sumologic/values.yaml [hunk 2]
```diff
@@ -4348,7 +4348,7 @@ otellogs:
           - id: parser-crio
             type: regex_parser
             regex: '^(?P<time>[^ Z]+) (?P<stream>stdout|stderr) (?P<logtag>[^ ]*) (?P<log>.*)$'
-            output: extract-metadata-from-filepath
+            output: merge-cri-logs
```
Comment (pmalek-sumo) [inline deploy/helm/sumologic/values.yaml:4351]: Just as a refresher: Why the `output` is set to a name of another operator?

Comment (sumo-drosiek) [reply deploy/helm/sumologic/values.yaml:4351]: by default `output` is set to next operator in list, but we want to go to `merge-cri-logs` as next in pipeline

File: deploy/helm/sumologic/values.yaml (hunk 1 above)
Comment (swiatekm-sumo) [reply deploy/helm/sumologic/values.yaml:4408]: @astencel-sumo you wrote this originally, I just copied it mindlessly.

File: deploy/helm/sumologic/values.yaml (hunk 2 above)
Comment (swiatekm-sumo) [reply deploy/helm/sumologic/values.yaml:4351]: That's how operators for filelog receiver work. You explicitly set which operator the output should go to if it's not the next one in the list.

**Comment by sumo-drosiek** [comment]
Closing due to:

* #2155
* #2156
* #2158
* #2159
* #2160
* #2161
* #2162

**Pull request closed without merging**